docs/swagger.yaml

```diff
@@ -180,6 +180,134 @@ paths:
         - $ref: '#/components/parameters/Merchant-Serial-Number'
```

Comment:
Not a comment on this spesific line - but try to use task-numbers as branches (e.g. WE-501), if we do that the commits will line up with the task in jira and our change management :)

Reply:
I will create a new branch with the right name when this PR has been approved and all comments has been resolved. Then I will delete this branch, and push a new one with the right name. Create a new PR and then ask you guys to approve it.

Reply:
oh no worries, meant in the future 😅